src/storage/src/region/writer.rs

```diff
@@ -392,6 +392,7 @@ impl WriterInner {
         mut request: WriteBatch,
         writer_ctx: WriterContext<'_, S>,
     ) -> Result<WriteResponse> {
+        let _timer = timer!(crate::metrics::LOG_STORE_WRITE_ELAPSED);
```

Comment:
This timer measures the whole write time of the engine instead the logstore.

You might need to rename it to the region write elapsed or move the timer to `write_to_wal`
https://github.com/GreptimeTeam/greptimedb/blob/4ae0b5e1855ba37e0007e9f0c55fc5577b200c06/src/storage/src/wal.rs#L103